Sources/PointFree/Stripe.swift

```diff
@@ -244,17 +246,6 @@ private func fetchSubscription(id: Stripe.Subscription.Id) -> EitherIO<Prelude.U
   return stripeDataTask("subscriptions/" + id.unwrap + "?expand[]=customer")
 }
 
-private func reactivateSubscription(_ subscription: Stripe.Subscription)
-  -> EitherIO<Prelude.Unit, Stripe.Subscription> {
-
-    guard
-      subscription.cancelAtPeriodEnd,
-      let item = subscription.items.data.first
-      else { return throwE(unit) }
```

Comment:
This `guard` is already fulfilled with a filterMap, so just using `updateSubscription` from the reactivation middleware.